tests/ztest/base/src/main.c

```diff
@@ -36,6 +36,23 @@ ZTEST(framework_tests, test_assert_mem_equal)
 	zassert_mem_equal(actual, expected, sizeof(expected), NULL);
 }
 
+ZTEST(framework_tests, test_assert_str_equal)
+{
+	const char *s1 = "asdf";
+	const char s2[] = {'a', 's', 'd', 'f', '\0'};
+
+	zassert_str_equal(s1, s2, NULL);
```

Comment:
We don't need the third argument here, right? The point of `, ##__VA_ARGS__` is that it expands to nothing when `__VA_ARGS__` is empty.